README.md

```diff
@@ -1,64 +1,45 @@
-![Cactus Logo Color](./images/HL_Cactus_Logo_Color.png)
+![Cacti Logo Color](./images/HL_Cacti_Logo_Color.png)
 
-# Hyperledger Cactus
+# Hyperledger Cacti
 
- [![Open in Visual Studio Code](https://open.vscode.dev/badges/open-in-vscode.svg)](https://open.vscode.dev/hyperledger/cactus)
- ![license](https://img.shields.io/github/license/hyperledger/cactus) [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/4089/badge)](https://bestpractices.coreinfrastructure.org/projects/4089) [![codecov](https://codecov.io/gh/hyperledger/cactus/branch/main/graph/badge.svg?token=BJklIsqf1S)](https://codecov.io/gh/hyperledger/cactus)
- ![GitHub issues](https://img.shields.io/github/issues/hyperledger/cactus)
+Hyperledger Cacti is a multi-faceted pluggable interoperability framework to link networks built on heterogeneous distributed ledger and blockchain technologies and to run transactions spanning multiple networks. This project is the result of a merger of the [Weaver Lab](https://github.com/hyperledger-labs/weaver-dlt-interoperability) project with **Hyperledger Cactus**, which was subsequently renamed to **Cacti**. It draws on the cutting-edge technological features of both constituent projects to provide a common general purpose platform and toolkit for DLT interoperability. This was the first-of-a-kind merger of two systems, architecture and code bases, to create a new project, under the Hyperledger Foundation. See this [Hyperledger Foundation blog article](https://www.hyperledger.org/blog/2022/11/07/introducing-hyperledger-cacti-a-multi-faceted-pluggable-interoperability-framework) for more information about the merger.
```

Comment:
Could you remove the following sentence?
> This project is the result of a merger of the Weaver Lab project with Hyperledger Cactus, which was subsequently renamed to Cacti. It draws on the cutting-edge technological features of both constituent projects to provide a common general purpose platform and toolkit for DLT interoperability. This was the first-of-a-kind merger of two systems, architecture and code bases, to create a new project, under the Hyperledger Foundation. See this Hyperledger Foundation blog article for more information about the merger.

Cactus and Weaver were already merged.  So, we should describe the purpose of Cacti without the old history of Cactus and Weaver in order to tell our purpose to new users

Reply:
@takeutak This is actually an important line for the time being because the code bases are not integrated, and we want existing Cactus and Weaver users to find code and features they are familiar with.

We can move the line to a separate (history) file later, after the first round of integration. Until then, it serves a real purpose in my opinion.

Reply:
@VRamakrishna Thanks, I agree with you.  Of course, this description will be rearranged in Cactus v2-beta or later.